sdk/core/azure-core/vcpkg/CONTROL

```diff
@@ -5,9 +5,14 @@ Description: Microsoft Azure Core SDK for C++
   This library provides shared primitives, abstractions, and helpers for modern Azure SDK client libraries written in the C++.
 Homepage: https://github.com/Azure/azure-sdk-for-cpp/tree/master/sdk/core/azure-core
 Supports: !uwp
-Default-Features: curl
+Default-Features: http
+
+Feature: http
+Build-Depends: azure-core-cpp[core, curl], azure-core-cpp[core, curl, winhttp] (windows)
```

Comment:
What is `core`?

Reply:
It is the built-in feature in vcpkg, every package has it. Core, in VcPkg terms, is the package without any features.